tests/Resources/DatabaseDump.php

```diff
@@ -158,9 +159,40 @@ public function restoreTable(string $table): void
         $restoreCommand = $this->buildMySQLCommand('mysql', [$this->databaseName]);
         $restoreCommand .= ' < ' . escapeshellarg($dumpFile) . ' 2> /dev/null';
         $this->exec($restoreCommand);
+    }
 
+    private function cleanClassCache(string $className): void
+    {
         // Clean EntityManager cache
         Cache::clean(sprintf('objectmodel_%s_*', $className));
+        // Clear static cache of the ObjectModel class related to the table
+        $staticMethodCall = sprintf('%s::resetStaticCache', $className);
+        if (is_callable($staticMethodCall)) {
+            call_user_func($staticMethodCall);
+        }
+    }
+
+    private function getClassName(string $table): string
+    {
+        if ($table === 'lang') {
+            return 'Language';
```

Comment:
Would it be any better using ::class? (would get rid of "magic" strings at least) :thinking: 
```suggestion
            return Language::class;
```